pkgs/by-name/su/surge-XT/package.nix

```diff
@@ -66,11 +76,23 @@ stdenv.mkDerivation rec {
     ]
   );
 
-  # see https://github.com/NixOS/nixpkgs/pull/149487#issuecomment-991747333
+  patches = [ ./clap-option.diff ];
+
   postPatch = ''
+    # see https://github.com/NixOS/nixpkgs/pull/149487#issuecomment-991747333
     export XDG_DOCUMENTS_DIR=$(mktemp -d)
+    substituteInPlace CMakeLists.txt --replace-fail \
+      'cmake_minimum_required(VERSION 3.15)' \
+      'cmake_minimum_required(VERSION 4.0)'
+    substituteInPlace libs/libsamplerate/CMakeLists.txt --replace-fail \
+      'cmake_minimum_required(VERSION 3.1..3.18)' \
+      'cmake_minimum_required(VERSION 4.0)'
   '';
 
+  passthru.updateScript = gitUpdater {
+    inherit rev-prefix;
+  };
```

Comment:
Hmm if you want this, maybe set `passthru.rev-prefix`, and use `finalAttrs.finalPackage.passthru.rev-prefix` in `src`. In that case, please add a separate commit that only changes the expression to use `finalAttrs` - before the commit that fixes the package.